agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/onboard/secured/OnboardingService.java

```diff
@@ -29,4 +31,10 @@
    * @return -
    */
   String generateAuthorizationUrl(AuthorizationRequestParameters parameters);
+
+  /** @return The last error, if the last onboarding failed */
+  String getLastErrorAsString();
+
+  /** @return The last error, if the last onboarding failed as object read from the JSON String */
+  Optional<OnboardingError> getLastError();
```

Comment:
Same here, I would prefer `getOnboardingError` and `getLastError` as field accessor.

Reply:
Done